server/controllers/index.js

```diff
@@ -52,14 +52,51 @@ router.get('/auth/logout', auth, (req, res) => {
 
 /* Create a room (a single instance of a game) */
 router.get('/api/room/create', auth, (req, res) => {
-    createRoom(req.session.user, roomId);
-    res.status(200).json({roomId: roomId});
-    roomId++;
+    const { user } = req.session;
+    /* TODO: add check for when a room cannot be created and return an error */
```

Comment:
Is this done?

Reply:
I'm only checking if a player has already created a room, we haven't really decided in what situations they're not allowed to create one, so idk